createcoveragereport.sh

```diff
@@ -37,4 +37,8 @@ $REPORTGENERATOR \
    -targetdir:coverage/report \
    -verbosity:Error
 
-# TODO(benwu): upload coverage/report to public blob storage
+choco install codecov
+
+# Assume we've created the coverage file by this point. If we haven't, there should already have been an error.
+# Pass whatever parameters we recieved.
+codecov -f "coverage/coverage-filtered.xml" "$@"
```

Comment:
I don't think we want to do this unconditionally.

I'd suggest accepting an `--upload` parameter that specifies the flag name. (It would be nice to limit the codecov uninformative name of "flag" for the report name.)